modules/drivers/mongo/src/metabase/driver/mongo/query_processor.clj

```diff
@@ -46,9 +46,6 @@
 ;; this is just a very limited schema to make sure we're generating valid queries. We should expand it more in the
 ;; future
 
-(def ^:private $addFields
-  :$addFields)
-
```

Comment:
Operators are defined as strings. This removal produces consistent compilation results in modified test, ie. also `$addFields` is string in compiled query.